api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/service/v1/ValidationRulesService.java

```diff
@@ -118,4 +125,17 @@ public void updateStudentAgeColumns(SdcSchoolCollectionStudentEntity studentEnti
     studentEntity.setIsAdult(DOBUtil.isAdult(studentDOB));
     studentEntity.setIsSchoolAged(DOBUtil.isSchoolAged(studentDOB));
   }
+
+  public boolean hasNoEnrollmentHistory(StudentRuleData studentRuleData){
+      var student = studentRuleData.getSdcSchoolCollectionStudentEntity();
+      var school = studentRuleData.getSchool();
+      LocalDateTime startOfMonth;
+      if (student.getCreateDate() == null){
```

Comment:
This is not possible; student must have a create date.

Reply:
yep - makes sense. this was one of the issues we discussed on Friday.  I only added the use of the updateDate to accommodate the 2 failing tests (unrelated to what I was working on). The tests are "testUpdateAndValidateSdcSchoolCollectionStudent_StudentWithValidationErr_ShouldUpdateStatusToLOADEDAndReturnStatusOk()" and "testUpdateAndValidateSdcSchoolCollectionStudent_StudentWithValidationWarning_ShouldUpdateStatusToLOADEDAndReturnStatusOk()" found in controller\v1\SdcSchoolCollectionStudentControllerTest.java. It ends up throwing a null pointer error because a createDate has not yet been generated for the student when it hits this validation rule. Would you recommend I try to edit those tests? Unfortunately, changing how the createDate handled in those tests resulted in other errors but I'm happy to dig deeper into their functionality to figure out a fix.